custom/icds_reports/templates/icds_reports/icds_app/mobile/awc-reports.html

```diff
@@ -14,15 +14,83 @@
       <div ng-if="!$ctrl.isTabularDataDisplayed">
         <div ng-if="$ctrl.step === 'demographics'">
           <div class="bar-chart">
-            <h2 class="center pl-3">{$ $ctrl.data.chart[0].key $}</h2>
+            <p class="center pl-3 pt-3 mb-0 font-18 font-weight-600 font-montserrat">{$ $ctrl.data.chart[0].key $}</p>
             <nvd3 options="$ctrl.chartOptions" data="$ctrl.data.chart" class="with-3d-shadow with-transitions" id="mobile-bar-chart"></nvd3>
           </div>
         </div>
         <kpi data="$ctrl.data.kpi"></kpi>
+        <div ng-if="haveAccessToFeatures && $ctrl.step === 'pse'">
+          <!--AWC days open per week-->
+          <div class="bar-chart">
+            <p class="center pl-3 pt-3 mb-0 font-montserrat font-weight-600 font-18">{$ $ctrl.data.charts[0][0].key $}</p>
+            <nvd3 options="$ctrl.lineChartDaysPerWeekOptions" data="$ctrl.data.charts[0]" class="with-3d-shadow with-transitions" id="mobile-bar-chart"></nvd3>
+            <div class="d-flex justify-content-center mb-3">
+              <div class="d-flex align-items-center mr-3">
+                <div class="label-representation-dot olive-dot"></div>
+                <p class="mb-0 font-montserrat font-weight-600 font-16">Number of Days</p>
+              </div>
+              <div class="d-flex align-items-center">
+                <div class="label-representation-dot blue-dot"></div>
+                <p class="mb-0 font-montserrat font-weight-600 font-16">Week</p>
+              </div>
+            </div>
+          </div>
+
+          <!--line chart for daily attendance-->
+          <div class="bar-chart">
```

Comment:
should this be `"line-chart"`? Or maybe we can rename the class to `"mobile-chart"` to share it?

Reply:
I will rename it